app/views/home/_navbar.html.haml

```diff
@@ -7,13 +7,13 @@
         = link_to 'Home', root_path
       %ul.flex.items-center.flex-col.md:flex-row.md:space-x-5.w-full
         %li
-          = link_to 'plants', plants_path, class: class_names(current_page?(plants_path) ? 'active' : 'navlink')
+          = link_to 'Plants', plants_path, class: class_names(current_page?(plants_path) ? 'active' : 'navlink')
         %li
-          = link_to 'people', people_path, class: class_names(current_page?(people_path) ? 'active' : 'navlink')
+          = link_to 'People', people_path, class: class_names(current_page?(people_path) ? 'active' : 'navlink')
     
     .flex.items-center.gap-2
       - if user_signed_in?
-        .btn-primary= link_to "#{current_user.email}", profile_path(current_user)
+        .btn-primary= link_to "Hi, #{current_user.username}", profile_path(current_user)
```

Comment:
The essence of this change is to make the navbar look better some. It doesn't seem okay for it to just display the user's email address, Also the links have been capitalized, this was repeated in the table headers in the table partial created.
<img width="1273" alt="Screen Shot 2022-07-21 at 1 40 11 PM" src="https://user-images.githubusercontent.com/32690770/180215678-ded09f3c-b54b-4f91-a6b2-b0f728cf6e24.png">